geopandas/array.py

```diff
@@ -296,6 +296,36 @@ def sindex(self):
             self._sindex = get_sindex_class()(self.data)
         return self._sindex
 
+    def sindex_generated(self):
+        """Use the `.sindex` attribute on a GeoDataFrame or GeoSeries
+        to generate a spatial index if it does not yet exist,
+        which may take considerable time based on the underlying index
+        implementation.  This function quickly checks
+        the existence of the index without generating the index.
```

Comment:
```suggestion
        """Check the existence of the spatial index without generating it.
        
        Use the `.sindex` attribute on a GeoDataFrame or GeoSeries
        to generate a spatial index if it does not yet exist,
        which may take considerable time based on the underlying index
        implementation.
```

Minor rewording to have the important bit describing the function first.